yb-voyager/cmd/importSchema.go

```diff
@@ -83,7 +84,12 @@ func importSchema() {
 		tdb = tgtdb.NewTargetDB(&tconf)
 		err = tdb.Init()
 		if err != nil {
-			utils.ErrExit("Failed to initialize the target DB: %s", err)
+			utils.ErrExit("failed to initialize the target DB: %s", err)
+		}
+
+		invalidTargetIndexesCache, err = tdb.InvalidIndexes()
```

Comment:
You can move this statement at `importData.go:1110` with a surrounding condition to only execute once:
```
if invalidTargetIndexesCache == nil {
  invalidTargetIndexesCache, err = tdb.InvalidIndexes()
  if err != nil {
    ...
  }
}
```